docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md

```diff
@@ -66,34 +114,26 @@ To configure a Google Workspace Source:
-
-1. Provide the **Client Email**, **Private Key**, and **Token URL** you got in the JSON file after you created service account credentials in the [setup section](#set-up-service-account-credentials) above.
-
-1. Provide the same **Scope** you defined for your service account during the [setup section](#set-up-service-account-credentials) above.
-
+1. The **Delegated User Email** is the email address of the user you want to call the API on behalf of. This user should have the necessary [permissions](https://support.google.com/a/answer/7519580?hl=en) to view the details of other users in your Google Workspace domain, such as an Admin role. At a minimum, the user should have the `Users:Read permission`.
+   Learn more about Domain-Wide Delegation of Authority:
+    * [Domain-Wide Delegation of Authority](https://developers.google.com/identity/protocols/oauth2/service-account#delegatingauthority).
+    * [Google's Delegate domain-wide authority to your service account document](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#delegate_domain-wide_authority_to_your_service_account).
+1. Provide the **Client Email**, **Private Key**, and **Token URL** you got in the JSON file after you created service account credentials in the [Creating Service Account](#creating-service-account) above.
+1. Provide the same **Scope** you defined for your service account during the [Adding OAuth Scope](#adding-oauth-scope) above.
 1. The **Query Parameter for Collecting User Data** section provides two options, **Customer** or **Domain Parameter**. See the [Directory API documentation](https://developers.google.com/admin-sdk/directory/reference/rest/v1/users/list#query-parameters) from Google for details.
-  * **Customer ID**:** The unique ID for the customer's Google Workspace account. In the case of a multi-domain account, to fetch all groups for a customer, fill this field instead of domain. You can also use the `my_customer` alias to represent your account's `customerId`. The `customerId` is also returned as part of the [Users resource](https://developers.google.com/admin-sdk/directory/v1/reference/users).
+  * **Customer ID**. The unique ID for the customer's Google Workspace account. In the case of a multi-domain account, to fetch all groups for a customer, fill this field instead of domain. You can also use the `my_customer` alias to represent your account's `customerId`. The `customerId` is also returned as part of the [Users resource](https://developers.google.com/admin-sdk/directory/v1/reference/users).
   * **Domain**: The domain name. Use this field to get fields from only one domain. To return all domains for a customer account, use the `customer` query parameter instead.
-
-1. When you are finished configuring the Source, click **Submit**.
+12. When you are finished configuring the Source, click **Submit**.
```

Comment:
I think the numbered steps may need to be fixed in this section.

Reply:
Oh, I tried to fix that, but it looks like the change was not reflected.